src/utils/dig.js

```diff
@@ -63,12 +69,12 @@ export const handleDig = async ({ domain, types, short, provider }) => {
 
         // Keep adding rows until we reach Discord 4096 char limit
         for (const row of sourceRows) {
-            if (output([...finalRows, row]).length > 4096) break;
+            if (output([...finalRows, row]).length > max_length) break;
             finalRows.push(row);
         }
 
         // Render and return final rows
-        return output(finalRows);
+        return output(finalRows) + (cdflag ? DNSSEC_DISABLED_WARNING_MESSAGE : '');
```

Comment:
```suggestion
        return `${output(finalRows)}${cdflag
          ? `\n${DNSSEC_DISABLED_WARNING_MESSAGE}`
          : ''}`;
```
Single linebreak here as the codeblock for results already has some padding at the bottom